mesos-client/build.gradle

```diff
@@ -0,0 +1,16 @@
+plugins {
+    id 'application'
+}
+
+dependencies {
+    implementation 'com.typesafe.akka:akka-stream_2.12:2.5.19'
+    implementation 'org.apache.mesos:mesos:1.7.1'
```

Comment:
Why not `implementation 'org.apache.mesos:mesos:1.7.1:shaded-protobuf'`? Do we have other dependencies?

Reply:
when I do that, I get e.g. this build failure 🤔 

```MesosCalls.scala:166: type mismatch;
 found   : ByteString (in com.google.protobuf) 
 required: ByteString (in org.apache.mesos.protobuf) 
      .setAcknowledge(Call.Acknowledge.newBuilder().setAgentId(agentId).setTaskId(taskId).setUuid(uuid).build())```

Reply:
So the import is wrong? Not sure why this should be different.